src/cryptographic_primitives/proofs/sigma_correct_homomorphic_elgamal_enc.rs

```diff
@@ -42,7 +42,7 @@ pub struct HomoElGamalStatement<P> {
 }
 
 impl<P> HomoELGamalProof<P>
-where P: ECPoint + PartialEq + Clone + Zeroize,
```

Comment:
Why PartialEq is removed from P? Doesn't it lose its functionality?

Reply:
Any ECPoint is required to be PartialEq. This is written in [trait constraints](https://github.com/ZenGo-X/curv/blob/d089ed3de56b144d9350c14d354207b8d3db6ee4/src/elliptic/curves/traits.rs#L30-L31)